client/app/telecom/pack/slots/hubic/activation/pack-hubic-activation.html

```diff
@@ -14,42 +14,23 @@ <h1 data-translate="hubic_activation_widget-title"></h1>
                  data-translate="hubic_activation_widget-intro">
             </div>
 
-	<div class="text-center" data-ng-if="PackHubicActivationCtrl.loading">
-		<oui-spinner></oui-spinner> 
-	</div>
             <div class="row">
                 <div class="col-xs-12 col-md-6">
-
-                    <div class="table-responsive" data-ng-if="!PackHubicActivation.loading && PackHubicActivation.hubicList.length">
-                        <table class="table table-hover table-condensed">
-                            <thead>
-                                <tr>
-                                    <th scope="col"
-                                        data-translate="hubic_activation_widget_hubic_header"></th>
-                                    <th scope="col"
-                                        data-translate="hubic_activation_widget_voucher_header"></th>
-                                </tr>
-                            </thead>
-
-                            <tbody>
-                            <tr class="active" data-ng-repeat="hubic in PackHubicActivation.hubicList track by hubic.domain">
-                                <th scope="row">
-                                    <strong data-ng-bind="hubic.domain"></strong>
-                                    ( <span data-ng-bind="hubic.size"></span> )
-                                </th>
-                                <td>
-                                    <span data-ng-if="hubic.voucher" data-ng-bind="hubic.voucher"></span>
-                                    <span data-ng-if="!hubic.voucher">-</span>
-                                </td>
-                            </tr>
-                            </tbody>
-                        </table>
-                    </div>
+                    <oui-datagrid data-rows="PackHubicActivation.hubicList"
+                                  data-ng-if="!PackHubicActivation.loading && PackHubicActivation.hubicList.length">
```

Comment:
Both conditions are handle by the datagrid, i'm would have prefer the default behaviour.